EIPS/eip-3779.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: 3779
+title: Safe Control Flow for the EVM
+status: Draft
+type: Standards Track
+category: Core
+author: Greg Colvin <greg@colvin.org>
```

Comment:
All EIPs should have at least one GitHub handle in the author list so we can be sure to have in-platform contact information for at least one person.  Either of the following are acceptable:
```suggestion
author: Greg Colvin (@gcolvin)
```
```suggestion
author: Greg Colvin (@gcolvin), Greg Colvin <greg@colvin.org>
```